### ISSUE TITLE
dead project ? 🙁

### ISSUE THREAD

**Issue by ludwig7685** [xTerradon/hcaptcha-solver#19]
is the project dead ?

**Comment by Shaeikh** [comment]
not really he probably got other stuff, or maybe working on v2, I don't think he will leave the project

**Comment by neuluis** [comment]
Hey, this is a personal project and I will not be working full-time on it. I will surely come back to it, but for now I will not put as much time in as I did before. I got somewhat far with the V2-Solving, to a point where around 30% of captchas captcha can be solved comfortably. Performance beyond that point could not be reached yet, which is why I did not implement any V2 Solving into the algorithm. I will attach a visualisation of the solving process below, the code is also in this repo under `dev/v2solver`. 

![v2_training](https://github.com/xTerradon/hcaptcha-solver/assets/64305142/4c88ceb7-726f-42b1-a48d-f524316d135c)

Feel free to work on the project and improve it if you like, for me this is more of a "proof-of-concept" and to sharpen my CNN skills. I believe there are many good captcha solvers out there already, maintained by teams of developers with more skill & time on their hands.
